x-pack/solutions/security/test/security_solution_api_integration/test_suites/siem_migrations/utils/es_queries_dashboards.ts

```diff
@@ -47,18 +48,27 @@ export const getDashboardsPerMigrationFromES = async ({
   });
 };
 
-export const deleteAllDashboardMigrations = async (es: Client): Promise<void> => {
-  await es.deleteByQuery({
-    index: [MIGRATIONS_INDEX_PATTERN],
-    query: {
-      match_all: {},
-    },
-    ignore_unavailable: true,
-    refresh: true,
+export const getDashboardResourcesPerMigrationFromES = async ({
+  es,
+  migrationId,
+}: {
+  es: Client;
+  migrationId: string;
+}) => {
+  return await es.search({
+    index: RESOURCES_INDEX_PATTERN,
+    size: 10000,
+    // query: {
```

Comment:
guessing you want to keep and uncomment this?

Reply:
🤦 . Thanks for pointing it out. Fixed in https://github.com/elastic/kibana/pull/232218/commits/b2c898bbb4112d19f57f9038b89d111e7ba0c86c